### ISSUE TITLE
Dual ethereum-pool-server FRONTEND (ETH & ETC) in the same machine

### ISSUE THREAD

**Issue by millenniumcm** [sammy007/open-ethereum-pool#381]
Hi guys,  i am trying workout dual open-ethereum-pool in the same machine (ETH & ETC),  ETH (sammy007)  and ETC (LeChuckDE),    all services are working  (dual redis, Nginx, geth & open-ethereum...),    also Nginx  is setup correctly for each services working each DNS (ETH  & ETC).... but  after update and  rebuild ./build.sh   the ETC Fronted,  it still  continue showing "**Stats API Temporarily Down**" error message.... ETH its working perfectly , both services & process and working also

#### System information

OS & Version:  Ubuntu 16.04 LTS

#### Expected behaviour

Working dual Fronted (ETH  & ETC open-ethereum-pool) in the same machine

#### Actual behaviour

Only working correctly 1 Frontend (ETH),  another one (ETC) is showing "**Stats API Temporarily Down**" message

#### Backtrace

````
[backtrace]
````

**Comment by phucvinh52** [comment]
You can add domain for each API.

**Comment by millenniumcm** [comment]
Yes, domain for each API its added and working,   its work differents domain (ETH & ETC) for the same machine, that its not the issue.... the matter is  with ETC frontend  (with ETH everything is working) that is not working properly and shows  "**Stats API Temporarily Down**"   (of course ETC blockchain is full sync and i have make rebuild ./build.sh  at each  update )

**Comment by mikeyb** [comment]
https://github.com/sammy007/open-ethereum-pool/issues/375 you already asked this and already fixed it

**Comment by millenniumcm** [comment]
No,  its a different ISSUE,  services (redis, nginx, geth....) are working... but ETC Frontend not.

**Comment by mikeyb** [comment]
It is not an issue with this software.  Read the instructions until it makes sense.  You seem to not understand how these services run, nevermind running them on the same machine.  Try building a machine just for ETC as we are not here to help you learn system administration.

If you have an actual bug to report, report it.  But the instructions on this repo are clear and easy to get working.

If you are in over your head, I can set all of this up for a fee.

**Comment by millenniumcm** [comment]
All services are running (dual DNS works in the same machine....) ......  and how much is the setup fee  ?

**Comment by minernl** [comment]
I saw the pool config in the other post.
can you send me the nginx config you run now.

**Comment by millenniumcm** [comment]
this is my nginx config ( i have modify the domain names, but its works perfectly)

This Nginx services

https://pastebin.com/u8kFd5Jn

When i launch......  myetcpool.com/api/stats   ..... appear

> {"candidatesTotal":0,"hashrate":0,"immatureTotal":0,"maturedTotal":0,"minersTotal":0,"nodes":[{"difficulty":"208661639536214","height":"6425576","lastBeat":"1535065710","name":"main"}],"now":1535065712291,"stats":{}}  <

Nginx-conf
-----------------------------------------------------

upstream eth {
  server 127.0.0.1:8080;
}
upstream etc {
  server 127.0.0.1:8081;
}
server {
  listen 80;
  server_name myethpool.dynu.net;
  root /home/poolserver/open-ethereum-pool/www/dist;
  index index.html index.htm;
  location / {
    try_files $uri $uri/ /index.html;
  }

  location /api {
    proxy_pass http://eth;
  }
}
server {
  listen 80;
  server_name myetcpool.dynu.net;
  root /home/poolserver/open-ethereum-classic/open-ethereumclassic-pool/www/dist;
  index index.html index.htm;
  location / {
    try_files $uri $uri/ /index.html;
  }
location /api {
                if ($request_uri ~* "/api/(.*)") {
                        proxy_pass  http://etc/apietc/$1;
                }
        }

}

**Comment by minernl** [comment]
change lines:
location /api {
 if ($request_uri ~* "/api/(.*)") {
 proxy_pass http://etc/apietc/$1;
 }
 }
into:
location /api {
 proxy_pass http://etc;
 }

**Comment by millenniumcm** [comment]
in ETC

without ......  if ($request_uri ~* "/api/(.*)") {     ...... do not appear myetcxxxx/api/stats ......... maybe the correct will be

**location /api {
if ($request_uri ~* "/api/(.*)") {
proxy_pass http://etc;**

??

**Comment by goodthebest** [comment]
@millenniumcm did you figure out how to fix API issue?
